_posts/2015-04-28-how-I-found-the-cheapest-place-to-live-in-austin.md

```diff
@@ -0,0 +1,274 @@
+---
+layout:    post
+title:     How I Found the Cheapest Place to Live in Austin
+date:      '2015-04-28T13:13:00-06:00'
+summary:   The full story of how I lived under $200 a month.
+category:  
+author:    max
+---
+
+Hi guys! I've spent three months at Austin and wanted to share the
+interesting road to finding a steady home. In retrospective, I lived in 4
+different locations and spent less than $200 overall. There are a lot of things
+I would have never thought possible to do just a couple months earlier, which
```

Comment:
There are a lot of things I would have never thought possible just a couple of months earlier. 

No hay necesidad de poner _to do_